core_v2/sources/v2_domains.move

```diff
@@ -81,12 +81,15 @@ module aptos_names_v2::v2_domains {
     /// Name is expired and out of grace period
     const ECANNOT_RENEW_NAME_THAT_IS_EXPIRED_AND_PAST_GRACE_PERIOD: u64 = 31;
 
+    #[resource_group(scope = global)]
+    struct ObjectGroup { }
+
     /// Tokens require a signer to create, so this is the signer for the collection
-    struct CollectionCapability has key, drop {
-        capability: SignerCapability,
+    struct App has key {
+        extend_ref: object::ExtendRef,
```

Comment:
nit: Import ExtendRef directly. Also name the struct something more meaningful than App